tests/tests_insights_client_register.yml

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Basic insights-client register/unregister
+  hosts: all
+  gather_facts: true
+  become: true
```

Comment:
you shouldn't put `become: true` here.  If you need to use `become` when you run the playbook locally (you shouldn't if you are using `tox -e qemu-ansible...` as in https://linux-system-roles.github.io/contribute.html, but perhaps you are running against another system), you can use `ansible-playbook -b`

Reply:
Ok I didn't knew this but I will run locally as you have indicated and remove this line